user_docs/en/userGuide.t2t

```diff
@@ -1447,8 +1447,14 @@ NVDA will ask if you really wish to do this.
 As with installing, NVDA must be restarted for the add-on to be fully  removed.
 Until you do, a status of "remove" will be shown for that add-on in the list.
 
+To enable or disable an add-on, select an add-on from the list and press Enable or Disable button.

```

Comment:
Perhaps:
To disable an add-on, press the disable button.
To enable a previously disabled add-on, press the enable button.